.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,7 +27,7 @@ If it fixes an open issue, please link to the issue here.
 
 ## How has this been tested?
 <!--
-Please describe in detail how you tested your changes.
+Please describe in detail how you tested your changes. If you have challenges writing tests, please consult the document linked. https://make.wordpress.org/core/handbook/testing/automated-testing/phpunit/ or reach the community on [slack](https://join.slack.com/t/classicpress/shared_invite/enQtNDIwNjY2OTg1MjAxLWJiM2U2NmY3ZjFlZjQ4Zjk2OGI4ZTg3NzY1ZTU3NzI3OTRjMTU0YzAzOWUyZmZlODgyOWE1YTViYjcwY2Y5YzI)
```

Comment:
Let's use https://www.classicpress.net/join-slack/ instead.

Reply:
Also I wonder if this should go in the main contributing document instead?

Reply:
good idea, better add there the petitions link instead (people start pr without check it)

Reply:
Petitions link doesn't make sense under "How has this been tested".   Better to point people to the contribution documentation which explains how the petitions work.

Reply:
Spot on

Reply:
Marking as unresolved, this link still needs an update.